pkgs/applications/networking/instant-messengers/cordless/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGoModule, fetchurl }:
+
+buildGoModule rec {
+  name = "cordless-${version}";
```

Comment:
Please use ``$pname``.

Reply:
To elaborate on this, by setting `pname = "cordless";` you can omit the `name = "cordless-${version}";` declaration as this is already implie set elsewhere for derivations.